.github/ISSUE_TEMPLATE/bug_report_port.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report (hardware / driver / port)
+about: Report an issue in a port or driver
+title: ''
+labels: ports
+assignees: ''
+
+---
+
+**For questions about MicroPython or for help using MicroPython, please see the MicroPython Forum -- https://forum.micropython.org/**
```

Comment:
There's probably no other way to do this? My main gripe with templates is that instead of just typing text I have to delete a bunch of things :)